### PR TITLE
9base: new package at 63916da

### PR DESCRIPTION
**Pull request by cemkeylan**
## Description of package

9base is a collection of Plan9 tools by suckless.org. It is much smaller than plan9port
as only the core utilities of Plan9 is installed.

## Checklist

- [X] Latest upstream version.
- [X] I agree to maintain this package (**required**).
- [X] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by cemkeylan** [comment]
I versioned it with the latest commit date, should I change that?

File: community/9base/version [hunk 1]
```diff
@@ -0,0 +1 @@
+20190911 1
```
Comment (dylanaraps) [inline community/9base/version]: The version should be the commit hash.

File: community/9base/sources [hunk 2]
```diff
@@ -0,0 +1 @@
+git+git://git.suckless.org/9base#63916da
```
Comment (dylanaraps) [inline community/9base/sources]: I don't know if we should prefer the longer hash or not.

File: community/9base/build [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+make
+make DESTDIR="$1" install
+
+# Since rc shell does not conflict with the
+# system tools, link it to /usr/bin
+mkdir -p "$1/usr/bin"
+ln -s /usr/local/plan9/bin/rc "$1/usr/bin/rc"
```
Comment (dylanaraps) [inline community/9base/build]: This can be removed.

**Comment by cemkeylan** [comment]
:+1:

**Pull request merged**